literature.bib

```diff
@@ -25299,96 +25352,120 @@ @manual{JML:manual
 @online{bluej,
 	title = {{Bluej IDE}},
 	author = {King's College London},
-	howpublished = {\url{https://www.bluej.org/}},
 	url = {https://www.bluej.org/},
-	date = {2020-01-06},
-	year = 2020,
-	note = {Accessed at January 06, 2020}
+	urldate = {2020-01-06},
+	year = 2020
 }
 
 @online{marlin,
 	title = {{Marlin Firmware}},
 	author = {Erik van der Zalm},
-	howpublished = {\url{http://marlinfw.org/}},
 	url = {http://marlinfw.org/},
-	date = {2019-12-02},
+	urldate = {2019-12-02},
 	year = 2019,
-	note = {Accessed at December 02, 2019}
 }
 
 @online{linux,
 	title = {{Linux Operating System}},
 	author = {Linus Torvalds},
-	howpublished = {\url{https://www.kernel.org/}},
 	url = {https://www.kernel.org/},
-	date = {2019-12-02},
+	urldate = {2019-12-02},
 	year = 2019,
-	note = {Accessed at December 02, 2019}
 }
 
 @online{apoweb,
 	author = {Dirk Aporius},
 	title = {{Apo-Games}},
 	year = 2020,
-	howpublished = {\url{http://apo-games.de/}},
 	url = {http://apo-games.de/},
-	date = {2020-06-25},
-	note = {Accessed at June 25, 2020}
+	urldate = {2020-06-25},
 }
 
-@misc{featureide,
+@online{featureide,
 	title = {{FeatureIDE Repository}},
-	howpublished = {\url{https://github.com/FeatureIDE/FeatureIDE}},
+	author = {FeatureIDE Development Team},
+	url = {https://github.com/FeatureIDE/FeatureIDE},
 	year = 2020,
-	note = {Accessed: 2020-12-22}
+	urldate = {2020-12-22}
 }
 
-@misc{splot:fmgen,
-	howpublished = {\url{http://52.32.1.180:8080/SPLOT/splot_fm_generator.html}},
-	note = {Accessed: 2020-08-23}
+@online{splot:fmgen,
+	title = {{SPLOT FM Generator}},
+	author = {SPLOT Development Team},
+	url = {http://52.32.1.180:8080/SPLOT/splot_fm_generator.html},
+	urldate = {2020-08-23},
+	year = 2020
 }
 
-@misc{ABCD, 
-	howpublished = {\url{http://fmv.jku.at/abcd/}},
-	note = {Accessed: 2020-10-25}
+@online{ABCD,
+	title = {{ABCD}},
+	author = {Armin Biere},
+	url = {http://fmv.jku.at/abcd/},
+	urldate = {2020-10-25},
+	th-tag = {BDD Library},
+	year = 2020
 }
 
-@misc{BeeDeeDee,
-	howpublished = {\url{https://github.com/JuliaSoft/BeeDeeDee}},
-	note = {Accessed: 2020-09-12},
-	th-tag = {BDD Library}
+@online{BeeDeeDee,
+	title = {{BeeDeeDee}},
+	author = {Julia s.r.l.},
+	url = {https://github.com/JuliaSoft/BeeDeeDee},
+	urldate = {2020-09-12},
+	th-tag = {BDD Library},
+	year = 2020
 }
 
-@misc{Biddy,    
-	howpublished = {\url{http://biddy.meolic.com/}},
-	note = {Accessed: 2020-10-24}
+@online{Biddy,
+	title = {{Biddy}},
+	author = {Robert Meolic},
+	url = {http://biddy.meolic.com/},
+	urldate = {2020-10-24},
+	th-tag = {BDD Library},
+	year = 2020
 }
 
-@misc{BuDDy,
-	howpublished = {\url{http://buddy.sourceforge.net/manual/main.html}},
-	note = {Accessed: 2020-03-02}
+@online{BuDDy,
+	title = {{BuDDy}},
+	author = {Jorn Lind-Nielsen},
+	url = {http://buddy.sourceforge.net/manual/main.html},
+	urldate = {2020-03-02},
+	th-tag = {BDD Library},
+	year = 2020
```

Comment:
`online` is also not recognized by bibtex. So this references just becomes `Jorn Lind-Nielsen. BuDDy, 2020.`

Reply:
Yeah, we should definitely use `@misc` here, as this will lead to problems once Reference Styling from ACM / Springer is applied.

Reply:
changed to `@misc`